in_progress/roles.md

```diff
@@ -0,0 +1,91 @@
+New core developers can be nominated by any existing core developer, and for details on that process see our core developer guide.
+All core packages must have at least one core developer as a maintainer.
+Primary maintainers of new core packages, who are not already core developers, will be invited to join.
+
+## Steering Council
+
+The Steering Council (SC) members are core developers who have additional responsibilities to ensure the smooth running of the project.
+SC members are expected to participate in strategic planning, approve changes to the governance model, and make decisions about funding granted to the project itself.
+(Funding to community members is theirs to pursue and manage). 
+The purpose of the SC is to ensure smooth progress from the big-picture perspective.
+Changes that impact the full project require analysis informed by long experience with both the project and the larger ecosystem.
+When the core developer community (including the SC members) fails to reach such a consensus in a reasonable timeframe, the SC is the entity that resolves the issue.
+
+Members of the steering council also have the "owner" (**TBD**) role within the [scverse GitHub organization](https://github.com/scverse/)
+and are ultimately responsible for managing the [scverse](https://github.com/scverse) GitHub account, the [@scverse](https://twitter.com/scanpy_team)
+twitter account, the [scverse website](https://scverse.org), and other similar scverse-owned resources.
+
+The steering council is currently fixed in size to three members.
+This number will be increased as our community grows and diversifies, but will always be an odd number to ensure a simple majority vote outcome is always possible. 
+The initial steering council of the `scverse` consists of
+
+* [Isaac Virshup](https://github.com/ivirshup)
+* [Danila Bredikhin](https://github.com/gtca)
+* [Lukas Heumos](https://github.com/Zethson)
+
+The SC membership is revisited every January.
+SC members who do not actively engage with the SC duties are expected to resign.
+New members are added by nomination by a core developer.
+Nominees should have demonstrated long-term, continued commitment to the project and its mission and values.
+A nomination will result in discussion that cannot take more than a month and then admission to the SC by consensus.
+During that time deadlocked votes of the SC will be postponed until the new member has joined and another vote can be held.
+
+The `scverse` steering council may be contacted at `steering-council@scverse.org`, or via the [@scverse/steering-council](https://github.com/orgs/scverse/teams/steering-council) GitHub team.
+
+# Decision Making Process
+
+Decisions about the future of the project are made through discussion with all members of the community.
+All non-sensitive project management discussion takes place on the issue trackers of the https://github.com/scverse repositories.
+Occasionally, sensitive discussion may occur via a private message.
```

Comment:
I trust that what's listed here is also what is done in other OSS orgs

Reply:
This document is based of the one used by `zarr`, `napari`, `skimage`, and `sklearn`